lib/disability_compensation/requests/form526_request_body.rb

```diff
@@ -180,9 +184,8 @@ class ReservesNationalGuardService
     attribute :unit_name, String
     attribute :unit_address, String
     attribute :component, String
-    attribute :title10_activation, Title10Activation
     attribute :unit_phone, UnitPhone
-    attribute :receiving_inactive_duty_training_pay, Boolean
+    attribute :receiving_inactive_duty_training_pay, String
```

Comment:
And similar question here with the `Boolean`. 

I see a comment further down 
> "YES", "NO", or nil

Is that true for this field as well? And if so, can you add a clarifying comment

Reply:
yes, it's the same.
i'll add that same comment :)